index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://kit.fontawesome.com/8c36836212.js" crossorigin="anonymous"></script>
```

Comment:
I think you should consider adding a title after the `<script>` tag on line 9, which  would improve your site's accessibility rating, SEO rating, and user experience. :smiley: